blog/MeetthehoodiesNickHehr.md

```diff
@@ -0,0 +1,33 @@
+---
+
+title: 'Meet the Hoodies: Nick Hehr'
+author: Pawan
+layout: blog-post
+comments: true
+
+---
+
+#Hello all! I recently had the great honor of interviewing Nick Hehr ([Twitter](https://twitter.com/hipsterbrown), [GitHub](https://github.com/hipsterbrown)). Let's see what he had to say:
```

Comment:
let’s make this a normal paragraph. the first paragraph is always bold in the blog. Sorry that I missed that before :(